apps/.storybook/main.js

```diff
@@ -0,0 +1,22 @@
+const storybookWebpackConfig = require('../webpack').storybookConfig;
+
+module.exports = {
+  stories: ['../src/**/*.story.@(js|jsx)'],
+  staticDirs: [
+    '../build/package/',
+    '../../dashboard/public',
+    '../../pegasus/sites.v3/code.org/public'
+  ],
+  addons: ['@storybook/addon-actions', '@storybook/addon-options'],
+  framework: '@storybook/react',
+  // TODO: Add webpack5 configuration below when we upgrade to webpack 5.
```

Comment:
Is this written down somewhere so we don't forget? Or will Storybook fully not work without this line uncommented

Reply:
it won't work! i have it written down personally since i'll be shipping the webpack upgrade (hopefully, unless something horrible comes up...) right after this is merged